### PR TITLE
Grant funding section

### PR DESCRIPTION
**Pull request by 1emu**
image

### CODE REVIEW

File: src/components/GrantRequest/GrantRequestSection.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  validated: boolean
+  formEdited: boolean
+  onBlur: () => void
+  children: React.ReactNode
+}
+
+function getSectionIcon(focused: boolean, formEdited: boolean, sectionNumber: number, validated: boolean) {
+  if (focused) {
+    return <GrantRequestSectionFocused sectionNumber={sectionNumber} />
+  } else {
```
Comment (andyesp) [inline src/components/GrantRequest/GrantRequestSection.tsx:25]: You can remove this else as you are returning in line 25.

File: src/components/GrantRequest/GrantRequestSection.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    if (!formEdited) {
+      return <GrantRequestSectionUnfocused sectionNumber={sectionNumber} />
+    } else {
```
Comment (andyesp) [inline src/components/GrantRequest/GrantRequestSection.tsx:28]: I think you can remove this else too

File: src/components/Icon/GrantRequestSection/GrantRequestSectionFocused.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+
+export interface GrantRequestSectionIconProps {
+  sectionNumber: number
+}
+
+export function getNumberIcon(sectionNumber: number, white?: boolean) {
+  const color = white ? 'white' : '#D2D0D5'
```
Comment (1emu) [inline src/components/Icon/GrantRequestSection/GrantRequestSectionFocused.tsx:8]: note to self: use var

File: src/intl/en.json [hunk 4]
```diff
@@ -153,6 +153,12 @@
       "server_invalid_status": "Error fetching catalyst statuses"
     },
     "grant": {
+      "funding": {
+        "invalid": "Funding is invalid",
+        "too_low": "Funding is too low. Minimum is 100 USD",
```
Comment (1emu) [inline src/intl/en.json:158]: this numbers could ve a variable, like:  `t('funding.too_low', {value: MIN_CONSTANT})`

File: src/pages/submit/grant.tsx [hunk 5]
```diff
@@ -73,9 +72,7 @@ export default function SubmitGrant() {
             <DecentralandLogo />
             <div>{t('page.submit_grant.title')}</div>
           </div>
-          <Button basic className="cancel">
-            {'Cancel'}
-          </Button>
+          <Button basic>{'Cancel'}</Button>
```
Comment (andyesp) [inline src/pages/submit/grant.tsx:75]: `i18n`?

File: src/entities/Grant/constants.ts [hunk 6]
```diff
@@ -3,10 +3,28 @@ import { GrantTierAttributes, GrantTierStatus, GrantTierType } from './types'
 export const GRANT_PROPOSAL_DURATION_IN_SECONDS = 1209600
 export const GRANT_PROPOSAL_MIN_BUDGET = 100
 export const GRANT_PROPOSAL_MAX_BUDGET = 240000
+export const MAX_LOWER_TIER_GRANT_FUNDING = 20000
+
+export const GRANT_PROPOSAL_MIN_PROJECT_DURATION = 1
+export const GRANT_PROPOSAL_MAX_PROJECT_DURATION = 12
+export const MIN_LOW_TIER_PROJECT_DURATION = GRANT_PROPOSAL_MIN_PROJECT_DURATION
```
Comment (andyesp) [inline src/entities/Grant/constants.ts:10]: I'd add GRANT_PROPOSAL_ to all these new variables. Or maybe drop that from the other ones 🤔

**Pull request closed without merging**